src/System.Diagnostics.Process/src/System/Diagnostics/ProcessStartInfo.cs

```diff
@@ -52,6 +53,16 @@ public ProcessStartInfo(string fileName, string arguments)
             _arguments = arguments;
         }
 
+        /// <devdoc>
+        ///     Specifies the name of the application that is to be started, as well as a set
+        ///     of command line arguments to pass to the application.
+        /// </devdoc>
+        public ProcessStartInfo(string fileName, IReadOnlyCollection<string> argumentList)
+        {
+            _fileName = fileName;
+            _argumentList = argumentList;
```

Comment:
What if argumentList is null?  Seems like we'll blow up later when we try to dereference startInfo.ArgumentList, since the getter is just returning the field.